<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.2.0 for Hugo"><meta name=author content="Silas Kieser"><meta name=description content="Mouse is the most used model for studying the impact of microbiota on its host, but the rep- ertoire of species from the mouse gut microbiome remains largely unknown. Accordingly, the similarity between human and mouse microbiomes at a low taxonomic level is not clear. We construct a comprehensive mouse microbiota genome (CMMG) catalog by assembling all currently available mouse gut metagenomes and combining them with published refer- ence and metagenome-assembled genomes. The 41’798 genomes cluster into 1’573 spe- cies, of which 78.1% are uncultured, and we discovered 226 new genera, seven new families, and one new order. CMMG enables an unprecedented coverage of the mouse gut microbiome exceeding 86%, increases the mapping rate over four-fold, and allows func- tional microbiota analyses of human and mouse linking them to the driver species. Compar- ing CMMG to microbiota from the unified human gastrointestinal genomes shows an overlap of 62% at the genus but only 10% at the species level, demonstrating that human and mouse gut microbiota are largely distinct. CMMG contains the most comprehensive col- lection of consistently functionally annotated species of the mouse and human microbiome to date, setting the ground for analysis of new and reanalysis of existing datasets at an unprecedented depth."><link rel=alternate hreflang=en-us href=https://silask.github.io/publication/cmmg/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#1565c0"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark media=print onload='this.media="all"' disabled><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.e6f7b650e2d80724e28cce23c674926c.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-111618908-1"></script>
<script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}function trackOutboundLink(e,t){gtag("event","click",{event_category:"outbound",event_label:e,transport_type:"beacon",event_callback:function(){t!=="_blank"&&(document.location=e)}}),console.debug("Outbound link clicked: "+e)}function onClickCallback(e){if(e.target.tagName!=="A"||e.target.host===window.location.host)return;trackOutboundLink(e.target,e.target.getAttribute("target"))}gtag("js",new Date),gtag("config","UA-111618908-1",{}),gtag("set",{cookie_flags:"SameSite=None;Secure"}),document.addEventListener("click",onClickCallback,!1)</script><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/media/icon_hu0144edc3649b1995b32e914262035870_346211_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu0144edc3649b1995b32e914262035870_346211_180x180_fill_lanczos_center_3.png><link rel=canonical href=https://silask.github.io/publication/cmmg/><meta property="twitter:card" content="summary_large_image"><meta property="og:site_name" content="Dr. Kieser"><meta property="og:url" content="https://silask.github.io/publication/cmmg/"><meta property="og:title" content="Comprehensive mouse microbiota genome catalog reveals major difference to its human counterpart | Dr. Kieser"><meta property="og:description" content="Mouse is the most used model for studying the impact of microbiota on its host, but the rep- ertoire of species from the mouse gut microbiome remains largely unknown. Accordingly, the similarity between human and mouse microbiomes at a low taxonomic level is not clear. We construct a comprehensive mouse microbiota genome (CMMG) catalog by assembling all currently available mouse gut metagenomes and combining them with published refer- ence and metagenome-assembled genomes. The 41’798 genomes cluster into 1’573 spe- cies, of which 78.1% are uncultured, and we discovered 226 new genera, seven new families, and one new order. CMMG enables an unprecedented coverage of the mouse gut microbiome exceeding 86%, increases the mapping rate over four-fold, and allows func- tional microbiota analyses of human and mouse linking them to the driver species. Compar- ing CMMG to microbiota from the unified human gastrointestinal genomes shows an overlap of 62% at the genus but only 10% at the species level, demonstrating that human and mouse gut microbiota are largely distinct. CMMG contains the most comprehensive col- lection of consistently functionally annotated species of the mouse and human microbiome to date, setting the ground for analysis of new and reanalysis of existing datasets at an unprecedented depth."><meta property="og:image" content="https://silask.github.io/publication/cmmg/featured.png"><meta property="twitter:image" content="https://silask.github.io/publication/cmmg/featured.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2017-01-01T00:00:00+00:00"><meta property="article:modified_time" content="2022-03-30T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://silask.github.io/publication/cmmg/"},"headline":"Comprehensive mouse microbiota genome catalog reveals major difference to its human counterpart","image":["https://silask.github.io/publication/cmmg/featured.png"],"datePublished":"2017-01-01T00:00:00Z","dateModified":"2022-03-30T00:00:00Z","author":{"@type":"Person","name":"Silas Kieser"},"publisher":{"@type":"Organization","name":"Dr. Kieser","logo":{"@type":"ImageObject","url":"https://silask.github.io/media/icon_hu0144edc3649b1995b32e914262035870_346211_192x192_fill_lanczos_center_3.png"}},"description":"Mouse is the most used model for studying the impact of microbiota on its host, but the rep- ertoire of species from the mouse gut microbiome remains largely unknown. Accordingly, the similarity between human and mouse microbiomes at a low taxonomic level is not clear. We construct a comprehensive mouse microbiota genome (CMMG) catalog by assembling all currently available mouse gut metagenomes and combining them with published refer- ence and metagenome-assembled genomes. The 41’798 genomes cluster into 1’573 spe- cies, of which 78.1% are uncultured, and we discovered 226 new genera, seven new families, and one new order. CMMG enables an unprecedented coverage of the mouse gut microbiome exceeding 86%, increases the mapping rate over four-fold, and allows func- tional microbiota analyses of human and mouse linking them to the driver species. Compar- ing CMMG to microbiota from the unified human gastrointestinal genomes shows an overlap of 62% at the genus but only 10% at the species level, demonstrating that human and mouse gut microbiota are largely distinct. CMMG contains the most comprehensive col- lection of consistently functionally annotated species of the mouse and human microbiome to date, setting the ground for analysis of new and reanalysis of existing datasets at an unprecedented depth."}</script><title>Comprehensive mouse microbiota genome catalog reveals major difference to its human counterpart | Dr. Kieser</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=e4daae985670cd29634c0ab837cb9c56><script src=/js/wowchemy-init.min.5c87d484a568cb9870aa36d295c02167.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Dr. Kieser</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Dr. Kieser</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#posts><span>Posts</span></a></li><li class=nav-item><a class=nav-link href=/#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li></ul></div></nav></div><div class=page-body><div class=pub><div class="article-container pt-3"><h1>Comprehensive mouse microbiota genome catalog reveals major difference to its human counterpart</h1><div class=article-metadata><div><span class=author-highlighted>Silas Kieser</span>, <span>Evgeny M. Zdobnov</span>, <span>Mirko Trajkovski</span></div><span class=article-date>March 2022</span></div><div class="btn-links mb-3"><a class="btn btn-outline-primary btn-page-header" href=https://github.com/SilasK/CMMG target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header" href=https://ezmeta.unige.ch/CMMG target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary btn-page-header" href=/project/atlas/>Project</a>
<a class="btn btn-outline-primary btn-page-header" href=https://doi.org/10.1371/JOURNAL.PCBI.1009947 target=_blank rel=noopener>DOI</a></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract>Mouse is the most used model for studying the impact of microbiota on its host, but the rep- ertoire of species from the mouse gut microbiome remains largely unknown. Accordingly, the similarity between human and mouse microbiomes at a low taxonomic level is not clear. We construct a comprehensive mouse microbiota genome (CMMG) catalog by assembling all currently available mouse gut metagenomes and combining them with published refer- ence and metagenome-assembled genomes. The 41’798 genomes cluster into 1’573 spe- cies, of which 78.1% are uncultured, and we discovered 226 new genera, seven new families, and one new order. CMMG enables an unprecedented coverage of the mouse gut microbiome exceeding 86%, increases the mapping rate over four-fold, and allows func- tional microbiota analyses of human and mouse linking them to the driver species. Compar- ing CMMG to microbiota from the unified human gastrointestinal genomes shows an overlap of 62% at the genus but only 10% at the species level, demonstrating that human and mouse gut microbiota are largely distinct. CMMG contains the most comprehensive col- lection of consistently functionally annotated species of the mouse and human microbiome to date, setting the ground for analysis of new and reanalysis of existing datasets at an unprecedented depth.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=/publication/#2>Journal article</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Publication</div><div class="col-12 col-md-9">PLOS Computational Biology</div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style><p><img src=graphical_abstract.tiff alt=graphical_abstract></p></div><div class=article-tags><a class="badge badge-light" href=/tag/metagenome-assembled-genomes/>Metagenome-assembled genomes</a>
<a class="badge badge-light" href=/tag/genome-resolved-metagenomics/>Genome-resolved metagenomics</a>
<a class="badge badge-light" href=/tag/metagenomics/>metagenomics</a>
<a class="badge badge-light" href=/tag/binning/>Binning</a>
<a class="badge badge-light" href=/tag/metagenome-atlas/>metagenome-atlas</a>
<a class="badge badge-light" href=/tag/virus/>Virus</a></div><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https://silask.github.io/publication/cmmg/&amp;text=Comprehensive%20mouse%20microbiota%20genome%20catalog%20reveals%20major%20difference%20to%20its%20human%20counterpart" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https://silask.github.io/publication/cmmg/&amp;t=Comprehensive%20mouse%20microbiota%20genome%20catalog%20reveals%20major%20difference%20to%20its%20human%20counterpart" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=Comprehensive%20mouse%20microbiota%20genome%20catalog%20reveals%20major%20difference%20to%20its%20human%20counterpart&amp;body=https://silask.github.io/publication/cmmg/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https://silask.github.io/publication/cmmg/&amp;title=Comprehensive%20mouse%20microbiota%20genome%20catalog%20reveals%20major%20difference%20to%20its%20human%20counterpart" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li></ul></div><div class="media author-card content-widget-hr"><a href=https://silask.github.io/><img class="avatar mr-3 avatar-circle" src="https://s.gravatar.com/avatar/591119f511660dde907dfd01eefeba71?s=200" alt="Silas Kieser"></a><div class=media-body><h5 class=card-title><a href=https://silask.github.io/>Silas Kieser</a></h5><h6 class=card-subtitle>Metagenome researcher</h6><p class=card-text>Husband, Father & Metagenome researcher</p><ul class=network-icon aria-hidden=true><li><a href=mailto:silas.kieser@unige.ch><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/SilasKieser target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href=https://github.com/silask target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href="https://scholar.google.ch/citations?user=73hLKTUAAAAJ" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li></ul></div></div><div class="article-widget content-widget-hr"><h3>Related</h3><ul><li><a href=/publication/atlas/>ATLAS: a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data</a></li><li><a href=/publication/nestlemd/>Antibiotic Treatment Leads to Fecal Escherichia coli and Coliphage Expansion in Severely Malnourished Diarrhea Patients</a></li><li><a href=/project/nestle/>Internship at Nestlé research</a></li><li><a href=/publication/nestlead/>Bangladeshi children with acute diarrhoea show faecal microbiomes with increased Streptococcus abundance, irrespective of diarrhoea aetiology</a></li></ul></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by>© 2023 Silas Kieser</p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script>
<script id=search-hit-fuse-template type=text/x-template>
        <div class="search-hit" id="summary-{{key}}">
          <div class="search-hit-content">
            <div class="search-hit-name">
              <a href="{{relpermalink}}">{{title}}</a>
              <div class="article-metadata search-hit-type">{{type}}</div>
              <p class="search-hit-description">{{snippet}}</p>
            </div>
          </div>
        </div>
      </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script>
<script src=/js/bootstrap.bundle.min.6aed84840afc03ab4d5750157f69c120.js></script>
<script src=/en/js/wowchemy.min.26bc5a5b73c468c9e767656a378ac5e3.js></script></body></html>