---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "3-day course on Metagenome-atlas"
event:
event_url: https://www.oulu.fi/uniogs/node/212131
location: virtual
address:
  street:
  city:
  region:
  postcode:
  country: Finnland
summary: |
  Metagenome-atlas Three commands to start analysing your metagenome data
abstract:


commentable: true
# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: 2021-09-03T22:50:40+02:00
date_end: 2021-10-04T17:00:00+02:00
all_day: false

# Schedule page publish date (NOT event date).
publishDate: 2021-10-03T22:50:40+02:00

authors: [admin]
tags: [teaching]

# Is this a featured event? (true/false)
featured: true

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
links:
  - name: Reactions
    url: https://twitter.com/SilasKieser/status/1445030136722739202
    icon_pack: hb
    icon: x

# Optional filename of your slides within your event's folder or a URL.
url_slides:

url_code:
url_pdf:
url_video:

# Markdown Slides (optional).
#   Associate this event with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: [atlas]
---

Metagenomics transformed how we study microbiomes by enabling the analysis of entire microbial communities from diverse environments, without the need for culturing. Recent improvements in algorithms enabled the assembly of larger and larger contigs from metagenomes, which then can be clustered together to produce draft genomes of new species. In this way, an almost unimaginable number of genomes from uncultured microbes could be assembled. These genomes allow for dataset-specific taxonomic analysis and genome-resolved functional annotation of metagenomes.

In this online course, we will familiarize the participants with the steps required in assembly-based metagenomics (assembly, binning, genome completeness estimation, taxonomic and functional annotation). In the hand-on session we will use metagenome-atlas, a pipeline that implements the commonly used tools in a single workflow allowing users to get started with only three commands. Finally, we will show how to use the output of metagenome-atlas to answer scientific questions. The course will also serve as a platform for interaction between researchers and students,  working with metagenome and metatranscriptome data.









## Slides
- [From reads to genomes](Slides_1_Reads_to_genomes.pdf)
- [Atlas in details](Slides_3_Atlas_detail.pdf)

For background reading have a look at my PhD thesis.

{{< cite page="/post/phd-thesis" view="3" >}}



## Program

**Schedule (three separate days)**
- Monday September 27
- Friday October 1
- Monday October 4
Time: 9:15 – 12:30 + 13:30 - 17:00 EEST



### Day 1

- What is metagenomics?
- From Reads to genomes
- :computer: [Workshop 1: Setup & Conda](https://metagenome-atlas.shinyapps.io/TryAtlas)




### Day 2
**Invited Speaker**:

**Dr. Sofie Thijs**, Center of Environmental Sciences, Hasselt University, Belgium


### Day 3

**Invited Speaker**:
> Unravelling phototrophic microbial mysteries in iron-rich ecosystems using integrated 'omics approaches

**Dr. Jackson M. Tsuji**, Japan Society for the Promotion of Science Postdoctoral Fellow, Hokkaido University, Japan

> How not to analyze a metagenome

**Dr. Katharina Kujala**, Water Resources and Environmental Engineering Research Unit, University of Oulu, Finland
